js/custom.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Created by arjun on 5/24/2016.
+ */
+
+var i=0;
+window.ready(
+    function ()
+{
+    var a = document.getElementByClassName("check");
+    
+    var ct = document.createElement("span");
+    ct.setAttribute("class","glyphicon");
+    ct.setAttribute("class","glyphicon-play-circle-");
+    ct.setAttribute("style","textAlign:center");
```

Comment:
No I am not using these at all these are all in comments. I tried using them for a completely different thing but I having issues implementing them the way I though I need to figure out something else.